src/lib/layer-manager.js

```diff
@@ -39,7 +39,7 @@ const LOG_PRIORITY_LIFECYCLE = 2;
 const LOG_PRIORITY_LIFECYCLE_MINOR = 4;
 
 export default class LayerManager {
-  constructor({gl}) {
+  constructor({gl, useDevicePixelRatio}) {
```

Comment:
> constructor({gl, useDevicePixelRatio}) {

Why is this the only arg that goes into the constructor? 
Can this only be changed on construction? (If so, that limitation should be documented).
Shouldn't this be set in some `LayerManager.setOptions` method that can be called whenever props change?